Library/Homebrew/Gemfile

```diff
@@ -23,7 +23,7 @@ end
 gem "activesupport"
 gem "concurrent-ruby"
 gem "mechanize"
-gem "patchelf" if ENV["HOMEBREW_PATCHELF_RB"]
+gem "patchelf", "~> 1.1" if ENV["HOMEBREW_PATCHELF_RB"]
```

Comment:
```suggestion
gem "patchelf" if ENV["HOMEBREW_PATCHELF_RB"]
```

Reply:
LGTM after this small change.

Reply:
I'm gonna go ahead and commit this, since it's already formatted as a suggestion.